backend/hitas/management/commands/hitasmigrate.py

```diff
@@ -25,8 +27,24 @@ def add_arguments(self, parser: CommandParser) -> None:
             help="Oracle database password (default 'oracle')."
             " Can be also set with 'HITAS_ORACLE_PASSWORD' environment variable.",
         )
+        # MacOS only:
+        # Before you can run the migration, you will need to install oracle instantclient:
+        # https://cx-oracle.readthedocs.io/en/latest/user_guide/installation.html#scripted-installation
+        parser.add_argument(
+            "--oracle-instantclient-path",
+            default="Downloads/instantclient_19_8",
+            help="Oracle instantclient location, MacOS only (default: 'Downloads/instantclient_19_8').",
+        )
 
     def handle(self, *args, **options) -> None:
+        if sys.platform.startswith("darwin"):  # MacOS
+            try:
+                lib_dir = os.path.join(os.environ.get("HOME"), options["oracle_instantclient_path"])
```

Comment:
[Pathlib](https://docs.python.org/3/library/pathlib.html) is the way to go nowadays, but this is fine as well, no need to change it.